src/node/markdown/markdown.ts

```diff
@@ -185,6 +185,10 @@ export interface MarkdownOptions extends Options {
    * @see https://vitepress.dev/guide/markdown#github-flavored-alerts
    */
   gfmAlerts?: boolean
+  /**
+   * @default 'Copy Code'
+   */
+  codeCopyButtonTitle?: string
```

Comment:
Not sure, but should I put this in `themeConfig` for it to be aligned with other similar options in terms of developer experience?